### ISSUE TITLE
supabase start fails - program "postgres" is needed by initdb but was not found in the same directory

### ISSUE THREAD

**Issue by csi-lk** [supabase/cli#2909]
**Describe the bug**

Pretty self explanitory, continues to restart with error

```
popen failure: Operation not permitted
initdb: error: program "postgres" is needed by initdb but was not found in the same directory as "/usr/bin/initdb"
```

**To Reproduce**
1. `supabase start`

**System information**
Rerun the failing command with `--create-ticket` flag.
 - Ticket ID: 1d365b4ac0b9406f8e5d657026224e0c
 - Version of OS: MacOS 14.6.1
 - Version of CLI: 1.223.10
 - Version of Docker: 20.10.8, build 3967b7d
 - Versions of services: (no access token provided)

**Comment by csi-lk** [comment]
Going back in time to CLI version `1.219.0` seems to work

**Comment by csi-lk** [comment]
Uninstalling and re-installing docker on my machine seems to have fixed this, closing

**Comment by prakashraman** [comment]
Yeah, I spent a few hours stuck at the same place. I then tried downgrading to `1.219.0`, I saw some progress but eventually that failed. I tried upgrading the docker, and still it did not work

I finally sorted to @csi-lk comment
> Uninstalling and re-installing docker on my machine seems to have fixed this, closing

This worked 🎉  ! I noticed that my docker desktop was completely new now, and looked similar to what is present in the supabase cli docs.
